concepticondata/conceptlists/Bowern-2012-348.tsv

```diff
@@ -0,0 +1,349 @@
+121	Bowern-2012-348-121	father's father	3018	FATHER'S FATHER	
+122	Bowern-2012-348-122	father's mother	3020	FATHER'S MOTHER	
+123	Bowern-2012-348-123	father's sister	170	FATHER'S SISTER	
+124	Bowern-2012-348-124	fear	1419	FEAR (BE AFRAID)	
+125	Bowern-2012-348-125	feather	1201	FEATHER	
+126	Bowern-2012-348-126	feces	675	FAECES (EXCREMENT)	
+127	Bowern-2012-348-127	fingernail	1258	FINGERNAIL	
+128	Bowern-2012-348-128	fire	221	FIRE	
+129	Bowern-2012-348-129	fire sticks	3022	FIRE STICK	
+130	Bowern-2012-348-130	fish	227	FISH	
+131	Bowern-2012-348-131	flat	1633	FLAT	
+132	Bowern-2012-348-132	flow	2003	FLOW	
+133	Bowern-2012-348-133	flower	239	FLOWER	
+134	Bowern-2012-348-134	fly	1441	FLY (MOVE THROUGH AIR)	
+135	Bowern-2012-348-135	fog	249	FOG	
+136	Bowern-2012-348-136	foot	1301	FOOT	
+137	Bowern-2012-348-137	forehead	123	FOREHEAD	
+138	Bowern-2012-348-138	forget	1523	FORGET	
+139	Bowern-2012-348-139	frog	503	FROG	
+140	Bowern-2012-348-140	fruit	1507	FRUIT	
+141	Bowern-2012-348-141	give	1447	GIVE	
+142	Bowern-2012-348-142	goanna	3023	GOANNA	
+143	Bowern-2012-348-143	good	1035	GOOD	
+144	Bowern-2012-348-144	grass	606	GRASS	
+145	Bowern-2012-348-145	grasshopper	607	GRASSHOPPER	
+146	Bowern-2012-348-146	grow	479	GROW	
+147	Bowern-2012-348-147	hair	1040	HAIR	
+148	Bowern-2012-348-148	hand	1277	HAND	
+149	Bowern-2012-348-149	happy	1495	HAPPY	
+150	Bowern-2012-348-150	he/she	2642	HE OR SHE	
+151	Bowern-2012-348-151	head	1256	HEAD	
+152	Bowern-2012-348-152	hear	1408	HEAR	
+153	Bowern-2012-348-153	heart	1223	HEART	
+154	Bowern-2012-348-154	heavy	1210	HEAVY	
+155	Bowern-2012-348-155	heel	980	HEEL	
+156	Bowern-2012-348-156	hide	602	HIDE (CONCEAL)	
+157	Bowern-2012-348-157	hill	614	HILL	
+158	Bowern-2012-348-158	hit (with hand)	1433	HIT	
+159	Bowern-2012-348-159	hold	1448	HOLD	
+160	Bowern-2012-348-160	honey	942	HONEY	
+161	Bowern-2012-348-161	hot	1286	HOT	
+162	Bowern-2012-348-162	house	1252	HOUSE	
+163	Bowern-2012-348-163	how	1239	HOW	
+164	Bowern-2012-348-164	hungry	1533	BE HUNGRY	
+165	Bowern-2012-348-165	husband	1200	HUSBAND	
+166	Bowern-2012-348-166	hut	1678	HUT	
+167	Bowern-2012-348-167	if	1459	IF	
+168	Bowern-2012-348-168	in/inside	1606	INSIDE	
+169	Bowern-2012-348-169	intestines	1991	INTESTINES	
+170	Bowern-2012-348-170	itch	148	ITCH	
+171	Bowern-2012-348-171	kangaroo/deer	2456	LARGE WILD HERBIVORE	
+172	Bowern-2012-348-172	kidney	489	KIDNEY	
+173	Bowern-2012-348-173	kill	1417	KILL	
+174	Bowern-2012-348-174	kiss	578	KISS	
+175	Bowern-2012-348-175	knee	1371	KNEE	
+176	Bowern-2012-348-176	know/be knowledgeable	1410	KNOW (SOMETHING)	
+177	Bowern-2012-348-177	kookaburra	2444	KINGFISHER	
+178	Bowern-2012-348-178	lake	624	LAKE	
+179	Bowern-2012-348-179	laugh	1355	LAUGH	
+180	Bowern-2012-348-180	leaf	628	LEAF	
+181	Bowern-2012-348-181	left	244	LEFT	
+182	Bowern-2012-348-182	leg	1297	LEG	
+183	Bowern-2012-348-183	lie down	215	LIE DOWN	
+184	Bowern-2012-348-184	light	630	LIGHT (RADIATION)	confirm that it is not a different LIGHT concept
```

Comment:
difficult but probably really the noun reading, unless you find "heavy" in the list as well?

Reply:
"HEAVY" is there, good to see it is the reasoning I had. In fact, now I come to think that it is probably "light" as "not heavy".